cudamapper/src/main.cu

```diff
@@ -370,15 +370,30 @@ void postprocess_and_write_thread_function(const int32_t device_id,
                                                 0.5);
             }
 
-            // write to output
+            // write to output PAF, SAM, or BAM format
             {
-                GW_NVTX_RANGE(profiler, "main::postprocess_and_write_thread::print_paf");
-                print_paf(overlaps,
-                          cigars,
-                          *application_parameters.query_parser,
-                          *application_parameters.target_parser,
-                          application_parameters.kmer_size,
-                          output_mutex);
+                GW_NVTX_RANGE(profiler, "main::postprocess_and_write_thread::print_function");
+                if (application_parameters.format == OutputFormat::PAF)
```

Comment:
Again, this `if` could also be put in `#ifdef GW_BUILD_HTSLIB`

Reply:
same comment as above, but fixed